docs/releases/upgrading/upgrade-v11.0.md

```diff
@@ -47,7 +47,7 @@ The first step is to edit the `<my-site>/Pipfile`:
 +++python_version = "3.9"
 ```
 
-Delete the `Pipfile.lock`.
+and delete the `Pipfile.lock`.
```

Comment:
I think having them as list points are more prominent to the users....We want to draw attention here! wdyt?